lib/ui/painting/display_list_deferred_image_gpu.h

```diff
@@ -122,15 +113,12 @@ class DlDeferredImageGPU final : public DlImage {
     sk_sp<SkImage> image_;
 
     ImageWrapper(const SkImageInfo& image_info,
-                 std::variant<sk_sp<DisplayList>, std::shared_ptr<LayerTree>>
-                     display_list_or_layer_tree,
+                 sk_sp<DisplayList> display_list,
                  fml::WeakPtr<SnapshotDelegate> snapshot_delegate,
                  fml::RefPtr<fml::TaskRunner> raster_task_runner,
-                 fml::RefPtr<SkiaUnrefQueue> unref_queue,
-                 uint32_t width,
-                 uint32_t height);
+                 fml::RefPtr<SkiaUnrefQueue> unref_queue);
 
-    void SnapshotDisplayList();
+    void SnapshotDisplayList(std::shared_ptr<LayerTree> layer_tree = nullptr);
```

Comment:
nit: add a doc comment about the parameter.

Reply:
Done